test-data/unit/check-typeddict.test

```diff
@@ -377,25 +377,18 @@ f(ll) # E: Argument 1 to "f" has incompatible type "List[TypedDict({'x': int, 'z
 from typing_extensions import Protocol
 from mypy_extensions import TypedDict
 
-class StrIntMap(Protocol):
-    def __getitem__(self, key: str) -> int: ...
+class StrObjectMap(Protocol):
+    def __getitem__(self, key: str) -> object: ...
 
 A = TypedDict('A', {'x': int, 'y': int})
 B = TypedDict('B', {'x': int, 'y': str})
 
-def fun(arg: StrIntMap) -> None: ...
+def fun(arg: StrObjectMap) -> None: ...
 a: A
 b: B
 fun(a)
-fun(b)  # Error
+fun(b)
```

Comment:
I would keep a test that a function that expects a protocol with `__getitem__ -> int` always fails.